saturn/environment.yml

```diff
@@ -18,33 +19,40 @@ dependencies:
 - dask-saturn==0.0.4
 - dask-xgboost
 - dask==2.20.0
+- datashader
 - distributed==2.20.0
 - fastparquet
+- firefox
```

Comment:
`firefox` is a _very_ heavy dependency - if there's a way around this (and `selenium`, that'd be very, very preferred

Reply:
Mentioned this in coffee but this requirement is coming from a save in bokeh. We can get around this by saving off a file with the data for the pie chart and making that plot on the fly rather than saving it.

Reply:
For the examples repo i actually removed the pie chart, so I think we are good to take them out!

Reply:
:fire: